pizzapi/payment.py

```diff
@@ -24,5 +24,5 @@ def find_type(self):
                     'DISCOVER': r'^6(?:011|5[0-9]{2})[0-9]{12}$',
                     'JCB': r'^(?:2131|1800|35\d{3})\d{11}$',
                     'ENROUTE': r'^(?:2014|2149)\d{11}$'}
-        return next((card_type for card_type, pattern in patterns.items()
+        return next((card_type for card_type, pattern in list(patterns.items())
```

Comment:
Why instantiate a list here?

Reply:
i think this was an automatic change with a built in python tool.  probably not needed, but havent tested it yet.